test/integration/operator_policy_complianceconfig_test.go

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test OperatorPolicy complianceConfig",
+	Ordered, Label("BVT"), func() {
+		const (
+			policyPath             = "../resources/operator_policy/test-op-complianceconfig.yaml"
+			policyPathModified     = "../resources/operator_policy/test-op-complianceconfig-modified.yaml"
+			policyName             = "test-op-complianceconfig"
+			operatorPolicyName     = "test-op-complianceconfig"
+			subscriptionNamespace  = "grcqeoptest-ns-48381"
+			policyPath1            = "../resources/operator_policy/test-op-complianceconfig-1.yaml"
+			policyPath1Modified    = "../resources/operator_policy/test-op-complianceconfig-1-modified.yaml"
+			policyName1            = "test-op-complianceconfig-1"
+			operatorPolicyName1    = "test-op-complianceconfig-1"
+			subscriptionNamespace1 = "grcqeoptest-ns-48382"
+			operatorVersionInitial = "quay-operator.v3.8.14"
+			operatorVersionRecent  = "quay-operator.v3.8.15" // This may need to be modified for newer versions
+		)
+
+		BeforeAll(func() {
+			By("Create namespace " + subscriptionNamespace)
+			_, err := common.OcManaged(
```

Comment:
OperatorPolicy will automatically create the namespace if it's missing, so I think this and the other namespace creation command can be removed.